seismic/inversion/wavefield_decomp/example_wf_continuation_tao.py

```diff
@@ -79,11 +80,12 @@ def example_4():
     rho = [rho_c]
     fixed_args = (flux_comp, mantle_props, Vp, rho, FLUX_WINDOW)
     H_min, H_max = (25.0, 45.0)
```

Comment:
The boundary is between 25 and 60 km deep beneath the continents and between 5 and 8 km deep beneath the ocean floor.

Reply:
This example problem is set up to give a good solution to demonstrate the methodology. When I am solving general, unknown cases I will use the broad range of depths you suggest.